app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command):
+    files = command.split(" ")[1::]
+    old_file = files[0]
+    new_file = files[1]
+    if old_file != new_file:
```

Comment:
You also should check the input command

Reply:
The function takes only one argument `command`, that is a string with command `cp`, file name to copy, and new file name, separated by spaces.